### PR TITLE
some small modifications

### PR DESCRIPTION
**Pull request by d1nd141**

### CODE REVIEW

**Comment by Felix3qH4** [comment]
Just saw your request, it will take some time tough as I have continued working too so I have to look at the differences and try to implement everything into what i did in the meantime.

File: documentation.md [hunk 1]
```diff
@@ -15,7 +15,14 @@ xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx<br>
...
 Where x is either a number or a lowercase letter.<br>
 
-This key has a limit of requests you can make to the API (per hour?) and for small projects this is enough, but if you really want to track all busses around Luxembourg, you will have to ask them to increase your limit.<br>
+This key has a limit of requests you can make to the API, for small projects this is enough, but if you really want to track all busses around Luxembourg, you will have to ask them to increase your limit.<br>
+
+Default limits are:
+    Hourly: 500 requests
+    Daily: 5000 requests
+
+There also exists a monthly and yearly quota, but the defined request amount is quite generous.<br>
```
Comment (Felix3qH4) [inline documentation.md:24]: :) I have a quota of 600'000 requests per day haha

Comment (Felix3qH4) [reply documentation.md:24]: what can you do with 5'000?
Can you even request all stations in Luxembourg?

Comment (Felix3qH4) [reply documentation.md:24]: ok, yes you can, there are 2786 stations in LUX

Comment (Felix3qH4) [reply documentation.md:24]: so twice a day you can call them all

**Comment by Felix3qH4** [comment]
@d1nd141 just open your next pull request when you are done with adding new content, if there are no conflicts, just push it yourself, I invited you into the project so you should have the rights to do it. If there are conflicts, try to resolve them or I can do it too.

**Comment by Felix3qH4** [comment]
And don't forget to update your fork as there were changes made.

File: documentation.md (hunk 1 above)
Comment (d1nd141) [reply documentation.md:24]: Atm i'm polling only every 5 mins Lux2Bascharage, and Bascharage2Lux trains to measure the real delay's.
Anyway, last week i noticed a train, delayed 10 mins on mobiliteit.lu, while the api gave 29 mins.....
I missed to check in cfl app.

You already tested https://data.public.lu/en/docapi/ ?
Swagger api, maybe useful.

Comment (Felix3qH4) [reply documentation.md:24]: Yes, that was the reason I created this script, to check how late they are.
I just don't get what you mean with the api? The link you provided is for the api of the data.public.lu website so it has nothing to do with mobility except that I can fetch the datasets for the schedules which I can faster by hand.

**Pull request merged**